From 8e53428e2542bd2ee11a02da3f1f04bfbdc1cae1 Mon Sep 17 00:00:00 2001
From: Fang Hui <b31070@freescale.com>
Date: Wed, 5 Nov 2014 09:56:42 +0800
Subject: [PATCH 368/602] MA-5837 - [sabresd_6dq]: support usb headset

Add AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET in primary audio.

Signed-off-by: Fang Hui <b31070@freescale.com>
---
 evk_6sl/audio_policy.conf        | 2 +-
 hdmidongle_6dq/audio_policy.conf | 2 +-
 imx53_smd/audio_policy.conf      | 2 +-
 sabreauto_6q/audio_policy.conf   | 2 +-
 sabreauto_6sx/audio_policy.conf  | 2 +-
 sabresd_6dq/audio_policy.conf    | 2 +-
 sabresd_6sx/audio_policy.conf    | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/evk_6sl/audio_policy.conf b/evk_6sl/audio_policy.conf
index c709621..60f7e4b 100644
--- a/evk_6sl/audio_policy.conf
+++ b/evk_6sl/audio_policy.conf
@@ -26,7 +26,7 @@ audio_hw_modules {
         sampling_rates 44100
         channel_masks AUDIO_CHANNEL_OUT_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL
+        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL|AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET
         flags AUDIO_OUTPUT_FLAG_PRIMARY
       }
       hdmi {
diff --git a/hdmidongle_6dq/audio_policy.conf b/hdmidongle_6dq/audio_policy.conf
index c709621..60f7e4b 100644
--- a/hdmidongle_6dq/audio_policy.conf
+++ b/hdmidongle_6dq/audio_policy.conf
@@ -26,7 +26,7 @@ audio_hw_modules {
         sampling_rates 44100
         channel_masks AUDIO_CHANNEL_OUT_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL
+        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL|AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET
         flags AUDIO_OUTPUT_FLAG_PRIMARY
       }
       hdmi {
diff --git a/imx53_smd/audio_policy.conf b/imx53_smd/audio_policy.conf
index 606e61f..dde94de 100644
--- a/imx53_smd/audio_policy.conf
+++ b/imx53_smd/audio_policy.conf
@@ -26,7 +26,7 @@ audio_hw_modules {
         sampling_rates 44100
         channel_masks AUDIO_CHANNEL_OUT_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_OUT_EARPIECE|AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL
+        devices AUDIO_DEVICE_OUT_EARPIECE|AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL|AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET
         flags AUDIO_OUTPUT_FLAG_PRIMARY
       }
       hdmi {
diff --git a/sabreauto_6q/audio_policy.conf b/sabreauto_6q/audio_policy.conf
index c28fc5f..65e5f5b 100644
--- a/sabreauto_6q/audio_policy.conf
+++ b/sabreauto_6q/audio_policy.conf
@@ -26,7 +26,7 @@ audio_hw_modules {
         sampling_rates 48000
         channel_masks AUDIO_CHANNEL_OUT_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL
+        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL|AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET
         flags AUDIO_OUTPUT_FLAG_PRIMARY
       }
       hdmi {
diff --git a/sabreauto_6sx/audio_policy.conf b/sabreauto_6sx/audio_policy.conf
index c28fc5f..65e5f5b 100644
--- a/sabreauto_6sx/audio_policy.conf
+++ b/sabreauto_6sx/audio_policy.conf
@@ -26,7 +26,7 @@ audio_hw_modules {
         sampling_rates 48000
         channel_masks AUDIO_CHANNEL_OUT_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL
+        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL|AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET
         flags AUDIO_OUTPUT_FLAG_PRIMARY
       }
       hdmi {
diff --git a/sabresd_6dq/audio_policy.conf b/sabresd_6dq/audio_policy.conf
index b61a0fe..48a6f96 100644
--- a/sabresd_6dq/audio_policy.conf
+++ b/sabresd_6dq/audio_policy.conf
@@ -26,7 +26,7 @@ audio_hw_modules {
         sampling_rates 44100
         channel_masks AUDIO_CHANNEL_OUT_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL
+        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL|AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET
         flags AUDIO_OUTPUT_FLAG_PRIMARY
       }
       hdmi {
diff --git a/sabresd_6sx/audio_policy.conf b/sabresd_6sx/audio_policy.conf
index b61a0fe..48a6f96 100644
--- a/sabresd_6sx/audio_policy.conf
+++ b/sabresd_6sx/audio_policy.conf
@@ -26,7 +26,7 @@ audio_hw_modules {
         sampling_rates 44100
         channel_masks AUDIO_CHANNEL_OUT_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL
+        devices AUDIO_DEVICE_OUT_SPEAKER|AUDIO_DEVICE_OUT_WIRED_HEADSET|AUDIO_DEVICE_OUT_WIRED_HEADPHONE|AUDIO_DEVICE_OUT_AUX_DIGITAL|AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET
         flags AUDIO_OUTPUT_FLAG_PRIMARY
       }
       hdmi {
-- 
1.8.0

